build.gradle

```diff
@@ -7,9 +7,16 @@ version '1.0'
 
 allprojects {
     apply plugin: 'io.spring.dependency-management'
+    apply plugin: 'checkstyle'
+
+    checkstyle {
+        config project.resources.text.fromUri('https://raw.githubusercontent.com/UvitaTeam/code-style/enhance/vivy-idea-plugin/src/main/resources/META-INF/file/checkstyle.xml')
```

Comment:
please use local file instead, you have it right next to the sources

Reply:
the idea is that each user will access the file remotely to enforce consistency with updates on the file and also since copying the project which contain the file is not needed.

Reply:
sure, but this project has own copy which is primary one :)